src/components/PreviewConcept/Concept.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { Remarkable } from 'remarkable';
+import {
+  NotionDialogContent,
+  NotionHeaderWithoutExitButton,
+  NotionDialogLicenses,
+  NotionDialogText,
+  NotionDialogImage,
+} from '@ndla/notion';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { VisualElement } from '../../interfaces';
+import { Concept as ConceptType } from '../SlateEditor/editorTypes';
+
+const StyledBody = styled.div`
+  margin: 0 ${spacing.normal} ${spacing.small};
+  .c-tabs {
+    margin-left: 0;
+    margin-right: 0;
+  }
+  .react-tabs__tab-panel--selected {
+    animation-name: fadeInLeft;
+    animation-duration: 500ms;
+  }
+`;
+
+interface Props {
+  concept: ConceptType;
+}
+
+const Concept: FC<Props & tType> = ({ concept }) => {
+  const markdown = new Remarkable({ breaks: true });
+
+  const renderMarkdown = (text: string) => {
+    const rendered = markdown.render(text);
+    return (
+      <>
+        <span dangerouslySetInnerHTML={{ __html: rendered }} />
+      </>
+    );
+  };
+
+  const VisualElement = () => {
+    const visualElement = concept.visualElement;
+    switch (visualElement?.resource) {
+      case 'image':
+        return (
+          <NotionDialogImage
+            alt={visualElement?.alt}
+            src={visualElement?.url}
```

Comment:
Bilder i visuelt element skal også ha støtte for crop og focuspoint så det må også med her.